ios/Toodle/Rust/ToodleLib.swift

```diff
@@ -119,14 +143,30 @@ class ToodleLib {
 
     func sync_now() -> Bool {
         let err = store_sync(self.raw, "00000000-0000-0000-0000-000000000117", "http://mentat.dev.lcip.org/mentatsync/0.1")
-        if let error = err.pointee.error {
+        if let error = err.pointee.err {
             let str = String(cString: error)
             print("Sync error \(str)")
             return false
         }
 
         return true
     }
+
+    func query(sql: String) -> Query {
+        return Query(raw: store_query(self.raw, sql))
+    }
+
+    func value(forAttribute attribute: String, onEntity entid: Int64) -> TypedValue? {
```

Comment:
This PR is a bit hard for me to follow 'cos it's not clear what parts of it are pure Toodle hackery and which parts of it you want to keep around. This seems like it should be the latter, but it prints to stdout and is sitting in a file with a bunch of commented-out stuff…